client/lib/api/observer_provider.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/widgets.dart';
```

Comment:
As requested, 0.0.2-nullsafety.0 published.  If you are going to copy this in instead, plz make sure to follow the license/noticing requirements.

Reply:
@advayDev1 - thanks. Just to make sure we get this right. Would the following notice in the commit message be sufficient for you or should we do anything different?

```
client/lib/api/observer_provider.dart copied under Apache 2.0 license from this original source (copyright @advayDev1 - Advay Mengle):
https://github.com/holvonix-open/observer-provider/blob/master/LICENSE
```

Reply:
@advayDev1 - thank you

Corrected Message below.... Just to make sure we get this right. Would the following notice in the commit message be sufficient for you or should we do anything different?

```
client/lib/api/observer_provider.dart copied under Apache 2.0 license from
this original source (copyright @advayDev1 - Advay Mengle):
https://github.com/holvonix-open/observer-provider/blob/master/lib/src/observer_provider.dart
```

Reply:
It's just a standard "NOTICE" - the original NOTICE from the original file can be copied into the third-party license notices where all the other ones are.  The code is actually open sourced by Holvonix as a company, not me individually.

Reply:
This is a placeholder - I plan on deleting it now that your package is upgraded to null safety.

Reply:
Returned to @advayDev1's package